go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/googlecloudplatform/gcsfuse
 go 1.18
 
 require (
-	cloud.google.com/go/storage v1.22.1
```

Comment:
please do go.mod changes in seperate cls.

Reply:
Cool. Then this CL will contain only the changes to integrate the dependency. Will create a new cl for the feature task and unit test.